satpy/tests/test_multiscene.py

```diff
@@ -453,6 +453,8 @@ def test_save_datasets_distributed(self):
         client_mock.compute.side_effect = lambda x: tuple(v for v in x)
         client_mock.gather.side_effect = lambda x: x
         future_mock = mock.MagicMock()
+        from dask.delayed import Delayed
+        future_mock.__class__ = Delayed
```

Comment:
I'm guessing this is needed for the split_results? Could you move the import higher up?

Reply:
Done.